pingtest.sh

```diff
@@ -1,18 +1,20 @@
 #! /bin/bash
 
+# Argument 1 ($1) is the server's url, and Argument 2 ($2)is the email address to which an error message should be sent
```

Comment:
i dont understand why you didn't really take my advice about comment conventions